src/parser.ts

```diff
@@ -191,41 +90,41 @@ export function create(
  * The parser options.
  */
 export interface Options {
-  // Allow module code
+  /** Allow module code */
```

Comment:
Guess this's also your editor/tooling here?

Reply:
Using `/** ... */` the comment will be registered by TypeScript as a TSDoc comment. Many editor has support for this. See https://github.com/microsoft/tsdoc

Reply:
👍